src/modules/Builder/components/ConditionRow/Condition/Condition.test.tsx

```diff
@@ -10,7 +10,8 @@ import {
   mockedSingleSelectFormValues,
   mockedSliderFormValues,
 } from 'shared/mock';
-import { getEntityKey, renderWithAppletFormData } from 'shared/utils';
+import { getEntityKey } from 'shared/utils';
+import { renderWithAppletFormData } from 'shared/utils/renderWithAppletFormData';
 import { ConditionRowType } from 'modules/Builder/types';
 import { ConditionType } from 'shared/consts';
 
```

Comment:
Please disable the warning in this file.

Reply:
what do mean? I propose to fix something different from this task in another thread, if it does not explicitly relate to the task